service/src/main/java/tds/assessment/services/impl/AssessmentAssembler.java

```diff
@@ -0,0 +1,69 @@
+package tds.assessment.services.impl;
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.Multimap;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import tds.assessment.Assessment;
+import tds.assessment.Form;
+import tds.assessment.Item;
+import tds.assessment.ItemConstraint;
+import tds.assessment.ItemProperty;
+import tds.assessment.Segment;
+import tds.assessment.Strand;
+
+/**
+ * A helper class for assembling the {@link tds.assessment.Assessment}
+ */
+class AssessmentAssembler {
```

Comment:
I believe it is being covered by the service calls.  That being said I believe that is incorrect.  This is my fault as I originally was the one that didn't right a unit test around the original mapper code.

Since 90% of the Assessment related mapping logic is within this particular class I think this should have its own unit test which tests the methods.  Then the service unit test should become much simpler because it'll just verify that the repositories are called and the mapper is called.  The last part may be a little tricky since these are all static which means they can't be mocked.